README.adoc

```diff
@@ -35,15 +31,6 @@ You can take a look at the link:sonar-project.properties[sonar-project.propertie
 - https://docs.sonarcloud.io/advanced-setup/languages/c-c-objective-c/#analysis-steps-using-compilation-database[Using compilation database (compile_commands.json)]
 - https://docs.sonarcloud.io/enriching/test-coverage/test-coverage-parameters/[Test coverage parameters]
 
-= Windows\MSBuild
```

Comment:
Why remove this section?

Reply:
@arseniy-sonar Because I do not see the point of having it on a project that should only demonstrate coverage and that has differences compared to the configuration projects that do not have the extra files.
What information is it supposed to carry for the user?

Reply:
I see, agreed, this section isn't relevant here, indeed.